.circleci/config.yml

```diff
@@ -0,0 +1,39 @@
+jobs:
+  build:
+    docker:
+      - image: cimg/python:3.9.2
+      - image: circleci/mysql:8.0.3
+        environment:
+          MYSQL_ROOT_PASSWORD: rootpw
+          MYSQL_DATABASE: test_db
+          MYSQL_USER: user
+          MYSQL_PASSWORD: passw0rd
+    steps:
+      - checkout
+      - run: python --version
+      - run:
+          name: Install dependencies
+          command: |
+            curl http://packages.stardog.com/stardog.gpg.pub | sudo apt-key add
+            echo "deb http://packages.stardog.com/deb/ stable main" | sudo tee -a /etc/apt/sources.list
```

Comment:
Why install stardog, and not just run this in docker? A simple docker-compose file will be easier to maintain than making sure you can install this properly.

Reply:
I will work on this on a next PR